site/content/docs/developing/sidecars.en.md

```diff
@@ -1,16 +1,20 @@
 # Sidecars
 Sidecars are additional containers that run along side the main container. They are usually used to perform peripheral tasks such as logging, configuration, or proxying requests.
 
+!!! Attention
+    Sidecars are not supported for Request-Driven Web Services.  
+
+!!! Attention
+    If your service runs on Windows OS, [FireLens](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_firelens.html), [AWS X-Ray](https://aws.amazon.com/xray/), and [AWS App Mesh](https://aws.amazon.com/app-mesh/) are not supported. Please check if your sidecar container enables Windows.
```

Comment:
Does "the service runs on Windows OS" mean the main container is using Windows image?

Reply:
Yes...will reword!